js/ui/html_editor/modules/toolbar.js

```diff
@@ -115,6 +126,8 @@ if(Quill) {
                 e.preventDefault();
             });
 
+            this._detectKeyboardFormatChanging();
```

Comment:
It's not a suitable name for me. 
1. We do not **detect** changes, we just subscribe to a keydown event to find out about some specific key is down later.
2. We detect not a format changing but a key press. Format is changed by ourself.

Mb smth like `_subscribeFormatHotKeys`?